ext/io/lib.rs

```diff
@@ -48,6 +48,13 @@ use winapi::um::processenv::GetStdHandle;
 #[cfg(windows)]
 use winapi::um::winbase;
 
+#[cfg(windows)]
+use std::sync::Arc;
+#[cfg(windows)]
+use std::sync::Condvar;
+#[cfg(windows)]
+use std::sync::Mutex;
```

Comment:
Use parking_lot::Mutex and parking_lot::Condvar to avoid unwraps? Doesn't look like poisoning is done here.